SVA/lib/mmu.c

```diff
@@ -1445,7 +1445,7 @@ sva_mm_load_pgtable (void * pg_ptr) {
      * Mark the page table pages as read-only again.
      */
 
-    sva_mm_flush_tlb(secmemp);
+    invltlb();
```

Comment:
Your modification is functionally correct, but you're making SVA call kernel code.  SVA does not trust kernel code, so we do not call into it (except for specific untrusted callbacks).

You need to write your own inline assembly code to flush the TLBs.  Writing it should be straightforward if you've done inline assembly code in C before.  If you haven't, please read up on it in the GCC documentation (search for Inline Assembly Constraints for GCC on Google).